types/ember-useragent/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for ember-useragent 0.12
+// Project: https://github.com/willviles/ember-useragent#readme
+// Definitions by: Cameron Dubas <https://github.com/camerondubas>
+//                 Chris Krycho <https://github.com/chriskrycho>
+//                 James C. Davis <https://github.com/jamescdavis>
+//                 Dan Freeman <https://github.com/dfreeman>
```

Comment:
```suggestion
//                 Chris Krycho <https://github.com/chriskrycho>
//                 James C. Davis <https://github.com/jamescdavis>
//                 Dan Freeman <https://github.com/dfreeman>
```
what about those Github users? I understand those are experts/commiters from Ember community, but AFAIK not directly involved into ember-useragent (according to commits history). We are trying to avoid adding people directly as the '@owner', due to owners involvment into automated communication (so at least with presumed consent)